river/river_test.go

```diff
@@ -221,6 +240,10 @@ func (s *riverTestSuite) TestRiver(c *C) {
 	c.Assert(r.Source["es_title"], Equals, "second 2")
 	c.Assert(r.Source["tenum"], Equals, "e3")
 	c.Assert(r.Source["tset"], Equals, "a,b,c")
+	c.Assert(reflect.DeepEqual(
```

Comment:
using `c.Assert(r.Source["es_mylist"], DeepEqual, []interface{}{"a", "b", "c"})` is better. :-)